CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 * [CHANGE] Use relative links from /ring page to make it work when used behind reverse proxy. #1896
 * [CHANGE] Deprecated `-distributor.limiter-reload-period` flag. #1766
 * [CHANGE] Ingesters now write only normalised tokens to the ring, although they can still read denormalised tokens used by other ingesters. `-ingester.normalise-tokens` is now deprecated, and ignored. If you want to switch back to using denormalised tokens, you need to downgrade to Cortex 0.4.0. Previous versions don't handle claiming tokens from normalised ingesters correctly. #1809
+* [CHANGE] Moved MinChunkAge to the Querier config as StoreMinQueryLookback, allowing the store to be skipped during query time if the metrics wouldn't be found.#1893 
+  * `store.min-chunk-age` has been removed
+  * `querier.store-min-query-lookback` has been added in it's place.
```

Comment:
This is again opaque tbh. `IngesterMaxQueryLookback` has the flag `querier.query-ingesters-within`, maybe renaming it to `querier.query-store-after` is better?

Reply:
@pracucci thoughts?

Reply:
> IngesterMaxQueryLookback has the flag querier.query-ingesters-within

I didn't notice the CLI flag for `IngesterMaxQueryLookback` was different. YAML config file name and CLI flags should always match, otherwise we introduce yet another inconsistency. If we wanna keep the `-querier.query-ingesters-within` naming, than I would:

1. Rename `IngesterMaxQueryLookback` to `QueryIngestersWithin` and set its YAML config name to `query_ingesters_within`
2. Then I agree with Goutham about naming it `-querier.query-store-after` and - as a consequence - name the YAML `query_store_after`

Reply:
Done!